src/plugins/slick.cellselectionmodel.ts

```diff
@@ -160,7 +160,7 @@ export class SlickCellSelectionModel {
     const colLn = this._grid.getColumns().length;
     const active = this._grid.getActiveCell();
     let dataLn = 0;
-    if (this._dataView) {
+    if (this._dataView && 'getPagingInfo' in this._dataView) {
```

Comment:
is this better than doing `if (typeof this._dataView?.getPagingInfo === 'function')`?

Reply:
not sure, probably just matter of style in this case. Type narrowing with `in` operator seems idiomatic in typescript (https://www.typescriptlang.org/docs/handbook/2/narrowing.html#the-in-operator-narrowing), not sure about performance characteristics (but it probably does not matter in this case).

Reply:
yeah I was just thinking about what this will be transpiled to, but I'm ok with this approach too